plugins/x86/x86_abi.ml

```diff
@@ -40,6 +49,10 @@ module SysV = struct
     | Arg 5 -> var r.(1)
     | Arg n -> stack Int.(n-6)
 
+  let arg = function
+    | `Basic {C.Type.Spec.t=`double} -> flt
```

Comment:
if I'm reading SysV AMD64 correctly, then `float` is passed in the same way as `double`. So, if we will extend our `flt` with an extra parameter for `width`, instead of hardcoding `64`, we can easily handle `float` as well.

Reply:
forgot to add the link to the standard, here it is, see pages 17-22
https://www.uclibc.org/docs/psABI-x86_64.pdf